tasks.py

```diff
@@ -84,12 +89,18 @@ def start(ctx, ports=SERVER_POOL_PORT, image=SRC_IMAGE_NAME, unsafe=False,
             {image} {command}".format(**cmd_params)
         )
 
+        while not os.path.isfile(CHECK_FILE_PATH):
+            print("** Checking code server status. Press Ctrl-C to exit **\r", end="")
+        print("** Code server is up and running successfully **")
+
+
 @task
 def stop(ctx, container=TARGET_CONTAINER_NAME, hide=True):
     result = ctx.run("sudo docker ps -q --filter='name={0}'".format(container))
     if result.stdout:
         print ("** Discarding the docker container <{0}>".format(container))
         ctx.run("sudo docker stop {0}".format(container))
         ctx.run("sudo docker rm {0}".format(container))
+        print ("** Done! Discarded the docker container <{0}>".format(container))
```

Comment:
You should also remove the CHECK_FILE on stop otherwise it will always find the fine even the next time you run the container.

Reply:
And you should remove it before you run the docker command in case that fails.

Reply:
Fixed